presto-orc/src/main/java/com/facebook/presto/orc/reader/SliceSelectiveStreamReader.java

```diff
@@ -151,4 +152,10 @@ public static int computeTruncatedLength(Slice slice, int offset, int length, in
         }
         return length;
     }
+
+    @VisibleForTesting
+    public void resetDataStream()
+    {
+        ((SliceDirectSelectiveStreamReader) directReader).resetDataStream();
```

Comment:
shouldn't resetDataStream be called on dictionaryReader as well? or on currentReader?

Reply:
@mbasmanova currentReader is only set after the first `startStripe()` is called. I added dictionaryReader too.

Reply:
```
public class SliceSelectiveStreamReader
        implements SelectiveStreamReader
{
   @VisibleForTesting
    public void resetDataStream()
    {
        ((SliceDirectSelectiveStreamReader) directReader).resetDataStream();
        ((SliceDictionarySelectiveReader) dictionaryReader).resetDataStream();
    }
}
```